src/main/java/org/cbio/gdcpipeline/reader/ClinicalReader.java

```diff
@@ -88,8 +92,13 @@ private void processCasesResponse() {
         JSONArray cases = GraphQLQueryUtil.getQueryList(gdcResponse, "cases");
         for (Object caseObject : cases) {
             GDCCase gdcCase = new GDCCase((JSONObject) caseObject, filterNormalSampleFlag);
-            this.clinicalDataModelList.add(new Sample(gdcCase));
-            this.clinicalDataModelList.add(new Patient(gdcCase));
+            clinicalDataModelList.add(new Sample(gdcCase));
+            clinicalDataModelList.add(new Patient(gdcCase));
+            for (String aliquotId : gdcCase.getAliquotIds()) {
+                caseIdToSampleId.put(aliquotId.toUpperCase(), gdcCase.getSampleId().toUpperCase());         
+            }
+            caseIdToSampleId.put(gdcCase.getCaseId().toUpperCase(), gdcCase.getSampleId().toUpperCase());
```

Comment:
so it is possible that a case id (patient id) has more than one sample. Can you separate the ID mappings to different maps? 

case id --> tcga patient id 
sample id --> patient id
aliquot id --> sample id 

and maybe one for patient id to list of sample ids? idk if that is necessary though but I think the separation of the 2 above will make it easier to tell where they're coming from, what level they are, etc.